content/reference/templates/rental.md

```diff
@@ -0,0 +1,13 @@
+# rental.liquid
+
+The `rental.liquid` template renders a detailed page for an individual rental.
+
+## Parameters
+
+Name             | Type             | Description
+-----------------|------------------|------------
+debug            | Boolean          | [Debug mode](introduction/index.html). Default = 0
+checkin          | Date (ISO 8601)  | Prefill booknow widget with checkin date
```

Comment:
`Can be used to prefill booknow widget with checkin date for example`

That's still adjustable to developers desires.